eng/Subsets.props

```
@@ -53,7 +53,7 @@
 
     <!-- Determine if we support running the .NET SDK on the target platform -->
     <_SdkToolsSupportedOS Condition="'$(TargetsMobile)' != 'true' and '$(TargetsLinuxBionic)' != 'true'">true</_SdkToolsSupportedOS>
-    <_SdkToolsSupportedArch Condition="'$(TargetArchitecture)' != 'armel'">true</_SdkToolsSupportedArch>
+    <_SdkToolsSupportedArch Condition="'$(TargetArchitecture)' != 'armel' and '$(TargetArchitecture)' != 'wasm'">true</_SdkToolsSupportedArch>
```

We now have a `TargetsWasm` property that should be used for this kind of check check, it makes adding `wasm64` easier:

```suggestion
    <_SdkToolsSupportedArch Condition="'$(TargetArchitecture)' != 'armel' and '$(TargetsWasm)' != 'true'">true</_SdkToolsSupportedArch>
```
(Applies to all other instances of this check too)